index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="scripts/main.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/mainStyle.css">
+    <title>Stephen's portfolio</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="toolbar">
+            <ul>
+                <li class="textPlaceholder"> <a href="#" aria-label="Ssenkooto Stephen"> Ssenkooto Stephen </a> </li>
+                <li> <button id="menuButton" type="button"> <img src="icons/menu.svg" alt=""> </button></li>
+                <li id="navigation">
+                    <ul class="container">
+                        <li class="textPlaceholder"><a href="#projects" aria-label="Portfolio"> Portfolio</a></li>
+                        <li class="textPlaceholder"><a href="#about" aria-label="About"> About</a></li>
+                        <li class="textPlaceholder"><a href="#contact" aria-label="Contact"> Contact</a></li>
+                        <li class="textPlaceholder"><a rel="noopener noreferrer" href="mailto:ssenkootos74@gmail.com"
+                                aria-label="Send Email"><img src="icons/mail.svg" alt="Send Email"></a></li>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Good job working on this toolbar. To start off with minimal exercise, the toolbar was supposed to only consist of only 2 sub-elements for now. An `a` tag and a `nav` tag

Secondly, your implementation for the navigation is good. But it wasn't needed for this task. You are required to only work on the toolbar and headline section of your chosen design.
When we move to the desktop version, the navigations will become handy.
The reason for this sequential app development is to help us as students understand the process of building a web app using the mobile-first approach.

I am suggesting you implement something like the example below:

```
<header>
     <a></a>
     <nav> // hamburger icon goes here </nav>
</header>
```